Sources/BetterSafariView/WebAuthenticationSession/WebAuthenticationPresenter.swift

```diff
@@ -179,7 +179,11 @@ extension WebAuthenticationPresenter {
             // MARK: ASWebAuthenticationPresentationContextProviding
             
             func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
-                return coordinator.view.window!
+                if #available(iOS 13, *, macOS 10.15, *) {
+                    ASPresentationAnchor()
+                } else {
+                    coordinator.view.window!
+                }
```

Comment:
```suggestion
                return coordinator.view.window ?? ASPresentationAnchor()
```
I think it would be good if existing window instance takes precedence over the new `ASPresentationAnchor` instance, as described in [Authenticating a User Through a Web Service](https://developer.apple.com/documentation/authenticationservices/authenticating_a_user_through_a_web_service).

Reply:
And a `return` keyword is missing here 😅